### PR TITLE
Add documentation for registering aliases in a plugin cli

### LINKED ISSUE #3: Add documentation for registering aliases in a plugin cli
With the recent [and-cli@1.3.1](https://github.com/AndcultureCode/AndcultureCode.Cli/releases/tag/v1.3.1) update to support aliases, we should have a section in here that showcases how to register aliases through the package.json file and through the command registry.

### PR DESCRIPTION
**Pull request by brandongregoryscott**
Fixes #3 Add documentation for registering aliases in a plugin cli

-   [x] Related GitHub issue(s) linked in PR description
-   [x] Destination branch merged, built and tested with your changes
-   [x] Code formatted and follows best practices and patterns
-   [-] Code builds cleanly (no _additional_ warnings or errors)
-   [x] Manually tested
-   [-] Automated tests are passing
-   [-] No _decreases_ in automated test coverage
-   [x] Documentation updated (readme, docs, comments, etc.)
-   [-] Localization: No hard-coded error messages in code files (_minimally_ in string constants)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +48,11 @@ $ ./plugin-cli.js
 
 #### Importing the base CLI and registering all of the base commands (`copy`, `dotnet`, `dotnet-test`, etc...)
 
+<details>
```
Comment (wintondeshong) [inline README.md:51]: Thought: Is there any way to indent the content of these given sections (ie. #### Importing the base CLI...) - right now it renders with these content areas actually outdented further than the headings.

Comment (wintondeshong) [reply README.md:51]: https://share.getcloudapp.com/L1uQ5bW6

**Review by wintondeshong — APPROVED** [review]
@brandongregoryscott aside from the thought around indenting. Looks great! this goes a long way for users

**Review by ralphmcintyre — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (brandongregoryscott) [reply README.md:51]: Hmm - I think we can indent the code within the block, but not the block itself. I did wrap these in collapsible panels so the overview of the readme is less dense. Is this better?

image

Comment (wintondeshong) [reply README.md:51]: Yeah, that's great 👍

**Pull request merged**